packages/cli/openapi-parser/src/schema/convertSchemas.ts

```diff
@@ -617,6 +618,10 @@ export function convertSchemaObject(
 }
 
 export function getSchemaIdFromReference(ref: OpenAPIV3.ReferenceObject): string {
+    if (ref.$ref.startsWith(SCHEMA_INLINE_REFERENCE_PREFIX)) {
+        return ref.$ref.replace(SCHEMA_INLINE_REFERENCE_PREFIX, "").split("/")[0] ?? "unknown";
```

Comment:
Not sure about falling back on `unknown` or if all the inline referenced schemas will actually conform to starting with `#/components/responses/`.

Reply:
yeah lets scope the `unknown` change to another PR, for now we should just throw